CHANGELOG.md

```diff
@@ -3,6 +3,108 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.3.0
+
+_Jan 21, 2022_
+
+A big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Allow to group rows based on column value (#3277) @flaviendelangle
+
+  ⚠️ This feature is temporarily available on the Pro plan until the release of the Premium plan.
+
+  To avoid future regression for users of the Pro plan, the feature needs to be explicitly activated using the rowGrouping experimental feature flag.
+
+  ```tsx
+  // To fully control
+  <DataGridPro
+    rowGroupingModel={rowGroupingModel}
+    onRowGroupingModel={newModel => setRowGroupingModel(newModel)}
+    experimentalFeatures={{ rowGrouping: true }}
+  />
+
+  // To initialize without controlling
+  <DataGridPro
+    initialState={{
+      rowGrouping: {
+        model: rowGroupingModel,
+      },
+    }}
+    experimentalFeatures={{ rowGrouping: true }}
+  />
+  ```
+
+  See the [documentation](https://mui.com/components/data-grid/group-pivot/#row-grouping) for more details.
+
+- ⚡ Add `is any of` filter operator (#2874) @alexfauquette
+- ✨ Introduce a `maxWidth` property in `GridColDef` (#3550) @flaviendelangle
+- 🙈 Add component slots for a subset of used `@mui/material` components (#3490) @DanailH
+
+  To make the grid more flexible we added component slots for base `@mui/material` components that we use. Those component slots are prefixed with `Base` to differentiate them from the other grid specific components
+
+  For more information check the documentation [documentation](https://mui.com/api/data-grid/data-grid/#slots).
+
+- 🔥 Allow to pass `csvOptions` and `printOptions` to `toolbar` component prop (#3623) @flaviendelangle
+
+  ```tsx
+  const CustomDataGrid = (props: DataGridProps) => {
+    return (
+      <DataGrid {...props} componentsProps={{ toolbar: { csvOptions: { delimiter: ';' }} />
+    )
+  }
+  ```
+
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v5.2.2` / `@mui/x-data-grid-pro@v5.2.2`
+
+#### Changes
+
+- [DataGrid] Add component slots for a subset of used `@mui/material` components (#3490) @DanailH
+- [DataGrid] Add controlled behavior for the visible columns (#3554) @flaviendelangle
```

Comment:
You missed the highlight for this one

Reply:
Added. Do we need an additional description or example?

Reply:
There is an example in the PR description
I think it's worth adding it.